### DISCUSSION TITLE
what to do ? "Permission denied (publickey)" error

### DISCUSSION THREAD

**Discussion by sebynooo** [fleetbase/fleetbase#166] [Q&A] [answered]
git clone git@github.com:fleetbase/fleetbase.git
Cloning into 'fleetbase'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by matheus-arruda** [comment, marked as answer]
The error occurs because you do not have SSH configured, try this:

`git clone https://github.com/fleetbase/fleetbase.git`

**Comment by qwerty314159265359** [comment]
Go to your profile and add the SSH Configuration
